contents/docs/revenue-analytics/start-here.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Getting started with revenue analytics
+sidebar: Docs
+hideAnchor: true
+---
+
+import DataWarehouseSources from './_snippets/data-warehouse-sources'
+import { QuestLog, QuestLogItem } from 'components/Docs/QuestLog'
+
+
+<QuestLog firstSpeechBubble="Let's get started!" lastSpeechBubble="Time to understand your revenue!">
+
+<QuestLogItem 
+    title="Choose your reporting currency"
```

Comment:
I feel like connecting a source should go first as it's the most important thing to do and probably has a much higher impact on activation/retention than reporting currency.